src/sentry/web/frontend/debug/mail.py

```diff
@@ -155,6 +159,46 @@ def render(self, request: Request):
         )
 
 
+@method_decorator(csrf_exempt, name="dispatch")
+class MailPreviewView(View, abc.ABC):
+    @abc.abstractmethod
+    def get_context(self, request):
+        pass
+
+    def get_subject(self, request):
+        return None
+
+    @property
+    @abc.abstractmethod
+    def html_template(self):
+        pass
+
+    @property
+    @abc.abstractmethod
+    def text_template(self):
+        pass
+
+    def get(self, request):
+        return MailPreview(
+            text_template=self.text_template,
+            html_template=self.html_template,
+            context=self.get_context(request),
+            subject=self.get_subject(request),
+        ).render(request)
+
+    def post(self, request):
+        msg = MessageBuilder(
+            subject=self.get_subject(request),
+            template=self.text_template,
+            html_template=self.html_template,
+            type="email.debug",
+            context=self.get_context(request),
+        )
+        msg.send_async(to=["dummy@stuff.com"])
```

Comment:
@Neo-Zhixing where is this email coming from?

Reply:
Just a placeholder for now. On sentry this gets printed on the console, on getsentry it'll open the system mail preview dialog.

Reply:
@Neo-Zhixing should it be a server setting then?

Reply:
No it should not. This address is just a placeholder. Debug views are only shown in dev environments. Neither of the two ways we show emails in dev environment cares about the recipient.